Source/Scene/ModelAnimationCollection.js

```diff
@@ -397,8 +398,9 @@ define([
                 scheduledAnimation._computedStartTime = JulianDate.addSeconds(defaultValue(scheduledAnimation.startTime, sceneTime), scheduledAnimation.delay, new JulianDate());
             }
 
-            if (!defined(scheduledAnimation._duration)) {
+            if (!defined(scheduledAnimation._duration) || scheduledAnimation._multiplierChanged) {
                 scheduledAnimation._duration = runtimeAnimation.stopTime * (1.0 / scheduledAnimation.multiplier);
+                scheduledAnimation._multiplierChanged = false;
```

Comment:
Should these be a separate if blocks?  Why do we need to update both `duration` and `multiplierChanged`?

Reply:
Is `multiplierChanged` actually used anywhere?  I can't find it

Reply:
See answer below https://github.com/AnalyticalGraphicsInc/cesium/pull/7361#discussion_r244353957